### ISSUE TITLE
Latex reader fails when \textit{} contains enumerate environment or empty lines

### ISSUE THREAD

**Issue by peterzjx** [jgm/pandoc#6833]
```
Error at "source" (line 1, column 9):
unexpected \begin
\textit{\begin{enumerate}\end{enumerate}}
```

My guess is it's related to https://github.com/jgm/pandoc/issues/5994 as in both cases there are block level elements. Hope this helps.

**Comment by jgm** [comment]
Yes, it's the same issue.  We could handle this kind of case by parsing the argument of `\textit` as blocks, then running a transformation that puts all `[Inline]` lists in the blocks into italics.  This kind of thing has got to be pretty rare, though, so I don't know if it's worth it.

**Comment by Delanii** [comment]
For this kind of thing I would suggest using switching commands like `\itshape` or `\bfseries`, that should solve your situation and is usable pretty much in any context (as far as I know).

**Comment by peterzjx** [comment]
I see. Thank you all for your answers. I am working on a project transforming latex from arxiv and indeed it is surprising to see how many rare and creative ways of using tex/latex people came up with.
